package.json

```diff
@@ -15,7 +15,8 @@
   },
   "scripts": {
     "docs": "shx rm -rf docs && typedoc --options typedoc.js",
-    "lint": "cross-env-shell \"tslint -t verbose --project . ${TSLINT_FLAGS}\"",
+    "lint": "eslint --max-warnings 0 \"**/*.{js,ts}\" && tslint -t verbose --project .",
+    "lint-fix": "eslint --max-warnings 0 \"**/*.{js,ts}\" --fix",
     "format": "prettier --write --loglevel warn \"./src/**/*.ts\"",
```

Comment:
`format` is entirely covered by `lint-fix` now, right?

Reply:
Should be yes, shall I remove?

Reply:
Make sense in my opinion. We can do the same for other repos step by step. But I think it is the right direction